src/util/map.test.js

```diff
@@ -294,4 +294,34 @@ describe("util/map", () => {
       ]);
     });
   });
+  describe("pushValue", () => {
+    it("works when the map has no matching key", () => {
+      const map = new Map();
+      MapUtil.pushValue(map, "foo", 3);
+      expect(map.get("foo")).toEqual([3]);
+    });
+    it("works when the map has a matching key", () => {
```

Comment:
This test is strictly weaker than “doesn’t add anything else to the
map”. Can we replace this test’s implementation with that test’s, and
delete the latter?

While we’re at it, we might as well replace the assertion on line 301
with `expect(map).toEqual(new Map().set("foo", [3]))`—this, too, looks
like a vestige of a former patch that used a new class instead of the
existing `Map` type.

Reply:
Done. Also modified the first test to follow the same pattern, and modified the second test to test its stated expectation more clearly.